src/Congo.RazorPages/Pages/Products.cshtml.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Congo.RazorPages.Models;
+using Congo.RazorPages.Services;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Congo.RazorPages.Pages
+{
+    public class ProductsModel : PageModel
+    {
+        public static List<Product> SampleData;
```

Comment:
This should be a non-static property

Reply:
We won't be modifying this list, so it can be changed to `IEnumerable`. You will need to do the same for `GetSampleProducts`